assets/sass/modules/_enroll.scss

```diff
@@ -283,7 +283,8 @@ $enroll-content-spacing: 15px;
 }
 
 .okta-verify-38 {
```

Comment:
can we rename this to something like `.okta-verify-download-icon`? it is no longer the 38x38 we had, so maybe this will help us know what each css class is for